README.md

```diff
@@ -21,29 +21,40 @@ _Xamarin Forms provides cross-platform APIs, but this project are only worked on
 
 The Tizen Wearable CircularUI project provides you the following UI controls:
 
+- ActionButtonItem : A class that presents a menu item and associates it with a command
+- BezelInteractionPage : A subclass of Xamarin.Forms.ContentPage that allows you to interact with Bezel rotation
 - Check : A subclass of Xamarin.Forms.Switch control supports Tizen specific style.
 - CircleDateTimeSelector : A control to select date or time fits in the circular screen.
+- CircleImage : A subclass of Xamarin.Forms.Image that holds an image with circle shape.
 - CircleListView : A subclass of Xamarin.Forms.ListView control fits in the circular screen and has the circular scrollbar.
 - CirclePage : A subclass of Xamarin.Forms.Page that can show Circular ProgressBar and Circular Slider and the button on the bottom of the screen. it can also show MenuItems on the circular menu.(MoreOption)
 - CircleProgressBarSurfaceItem : A control fits in the circular screen. it can be shown in CirclePage only.
-- CircleSliderSurfaceItem : A Slider control that responds to the bezel action and fits in the circular screen.
 - CircleScrollView : A subclass of Xamarin.Forms.ScrollView that can be scrolled by the bezel action.
+- CircleSliderSurfaceItem : A slider control that responds to the bezel action and fits in the circular screen.
 - CircleStackLayout : A container to layout children linear in the circular area.
 - CircleStepper : A control to select a number of steps fits in the circular screen.
 - CircleSurfaceEffectBehavior : The CircleSurfaceEffectBehavior is an effect which allows you to insert views that require CircleSurface.
-- ContextPopupEffectBehavior : the behavior to show a small popup has one or two button sticky with any control.
+- CircleSurfaceItem : A control to represent the items in the CircleSurface.
+- CircleSurfaceView : A subclass of Xamarin.Forms.View that can have several CircleSurfaceItem as a View not Page.
+- CircleToolbarItem : A subclass of Xamarin.Forms.ToolbarItem that can show toolbar item for fits in the circular screen.
+- CircularShell : A subclass of Xamarin.Forms.Shell that have additional properties for circular screen.
+- ColorMenuItem : A subclass of Xamarin.Forms.MenuItem that controls the background color of the button used in the popup control.
+- ContentButton : A button that allows you to customize the view to be displayed.
+- ContextPopupEffectBehavior : The behavior to show a small popup has one or two button sticky with any control.
```

Comment:
The explanation needs to be more clear..@_@

Reply:
Fixed.